yoojin_lee/LV_1/행렬의-덧셈.js

```diff
@@ -0,0 +1,11 @@
+function solution(arr1, arr2) {
+    let answer = [];
+    for (let i = 0; i < arr1.length; i++) {
+        let arr = [];
+        for (let j = 0; j< arr1[i].length; j++) {
+            arr.push(arr1[i][j] + arr2[i][j]);
+        }
+        answer.push(arr);
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
    const answer = [[]];
    for(let i = 0; i < arr1.length; i ++) {
        answer[i] = []
        for(let j = 0; j < arr1[i].length; j++) {
            answer[i].push(arr1[i][j] + arr2[i][j])
        }
    }
    return answer;
}
```

이렇게 answer 안에 빈 배열을 하나 더 둠으로써, 변수를 하나만 선언하고 해결할 수도 있습니다! for 문을 활용해서 잘 해결하셨네요👍

Reply:
오오 2차원 배열로 좀 더 효율적으로 작성할 수 있군요!!! 좋은 방법 공유 감사합니다~!!😊